### PR TITLE
Update Forkdiff

### PR DESCRIPTION
**Pull request by mininny**
Preview at https://strong-hedgehog.static.domains/

### CODE REVIEW

File: cl/persistence/state/minimal_state.go
Comment (ImTei) [inline cl/persistence/state/minimal_state.go:1]: Why did you delete these files?

Comment (mininny) [reply cl/persistence/state/minimal_state.go:1]: These files were deleted from upstream erigon, and are not modified on our end, which means they aren't necessary.

**Review by pcw109550 — CHANGES_REQUESTED** [review]
List of suggestions
- Can you remove `erigon-lib` hyperlink because it is not used anymore?
- Also ignore diffs for core/allocs/optimism_devnet.json because this diff adds +15277 line diffs but only needed for genesis allocs.

**Comment by mininny** [comment]
> List of suggestions
> 
> * Can you remove `erigon-lib` hyperlink because it is not used anymore?
> * Also ignore diffs for core/allocs/optimism_devnet.json because this diff adds +15277 line diffs but only needed for genesis allocs.

@pcw109550 Thanks for the suggestions! I removed `erigon-lib` and ignored the diffs for the .json file. 
What about `_test.go` files? What do you think about moving them all to the "Testing" section and have them ignored since they increase the diff so much? https://op-geth.optimism.io/ geth also has most of the tests in the Testing section as well.

**Comment by pcw109550** [comment]
@mininny ignoring test files will make forkdiff more consistent with op-geth forkdiff so i totally agree with that idea.

**Comment by mininny** [comment]
> @mininny ignoring test files will make forkdiff more consistent with op-geth forkdiff so i totally agree with that idea.

It's updated! :)

**Review by pcw109550 — APPROVED** [review]

**Pull request merged**